dts/arm/st/h5/stm32h562.dtsi

```diff
@@ -260,7 +260,7 @@
 
 		xspi1: spi@47001400 {
 			compatible = "st,stm32-xspi";
-			reg = <0x47001400 0x400>;
+			reg = <0x47001400 0x400>, <0x90000000 0x10000000>;
```

Comment:
Nit: use `DT_SIZE_K/M()` macros everywhere possible (at least for large numbers)

Reply:
done